src/main/java/software/amazon/awssdk/aws/greengrass/GreengrassCoreIPCService.java

```diff
@@ -17,7 +17,11 @@
 import software.amazon.awssdk.eventstreamrpc.EventStreamRPCServiceModel;
 import software.amazon.awssdk.eventstreamrpc.OperationContinuationHandlerContext;
 
+#if ANDROID
+public class GreengrassCoreIPCService extends EventStreamRPCServiceHandler {
+#else
 public final class GreengrassCoreIPCService extends EventStreamRPCServiceHandler {
+#endif
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;Invalid Character&#34;, @ expected

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.